docs/contributing.md

```diff
@@ -7,275 +7,256 @@ layout: single
-You can also run it anytime using:
+pre-commit will now run and check your code every time you make a commit. By default, it will only run on changed files,
+but you can run it on all files manually (this may be useful if you altered the pre-commit config):
 
 ```bash
 poetry run pre-commit run --all-files
 ```
 
-Your code must always be accompanied by corresponding tests, if tests are not present your code
-will not be merged.
-
 #### Pull requests
 
-* Fill in [the required template](https://github.com/python-poetry/poetry/blob/master/.github/PULL_REQUEST_TEMPLATE.md)
-* Be sure that your pull request contains tests that cover the changed or added code.
-* If your changes warrant a documentation change, the pull request must also update the documentation.
+* Fill out the pull request body completely and describe your changes as accurately as possible. The pull request body
+should be kept up to date as it will usually form the base for the final merge commit and the changelog entry.
+* Be sure that your pull request contains tests that cover the changed or added code. Tests are generally required for
+code be to be considered mergable, and code without passing tests will not be merged.
+* Ensure your pull requests passes the mypy and pre-commit checks pass. Remember that you can run these tools locally
+instead of relying on remote CI.
+* If your changes warrant a documentation change, the pull request must also update the documentation. Make sure to
+review the documentation preview generated by CI for any rendering issues.
 
 {{% note %}}
-Make sure your branch is [rebased](https://docs.github.com/en/free-pro-team@latest/github/using-git/about-git-rebase) against the latest main branch. A maintainer might ask you to ensure the branch is
-up-to-date prior to merging your Pull Request if changes have conflicts.
+Make sure your branch is [rebased](https://docs.github.com/en/get-started/using-git/about-git-rebase) against the latest
+base branch. A maintainer might ask you to ensure the branch is up-to-date prior to merging your pull request
+(especially if there have been CI changes on the base branch), and will also ask you to fix any conflicts.
 {{% /note %}}
 
-All pull requests, unless otherwise instructed, need to be first accepted into the main branch (`master`).
+All pull requests, unless otherwise instructed, need to be first accepted into the `master` branch. Maintainers will
+generally decide if any backports to other branches are required, and carry them out as needed.
 
 ### Issue triage
 
 {{% note %}}
-If you have an issue that hasn't had any attention, you can ping us `@python-poetry/triage` on the issue. Please, give us reasonable time to get to your issue first, spamming us with messages
+If you have an issue that hasn't had any attention, you can ping us `@python-poetry/triage` on the issue. Please give us
+reasonable time to get to your issue first, and avoid pinging any individuals directly, especially if they are not part
```

Comment:
```suggestion
reasonable time to get to your issue first, and avoid pinging any individual directly, especially if they are not part
```

Reply:
this is correct

Reply:
Oh ok, sorry, it sounded weird as a non-native English speaker. [TIL](https://english.stackexchange.com/questions/111444/any-individual-or-any-individuals) :slightly_smiling_face: